tools/stubs/phpunit-stub-defs.php

```diff
@@ -16,10 +16,10 @@
 }
 
 chdir( "$work_dir/phpunit" );
-if ( ! preg_match( '/^PHPUnit (\d+\.\d+\.\d+) /', (string) shell_exec( 'vendor/bin/phpunit --version' ), $m ) ) {
+$version = trim( (string) shell_exec( 'composer info phpunit/phpunit --format=json | jq -r \'.versions[0]\'' ) );
```

Comment:
Any reason you switched from `phpunit --version`?

Reply:
Because trying to run PHPUnit 12 with PHP 8.2 breaks. 😀

Reply:
Oh, of course. 😆